src/Symfony/Bridge/Twig/Extension/ImportMapRuntime.php

```diff
@@ -22,7 +22,7 @@ public function __construct(private readonly ImportMapRenderer $importMapRendere
     {
     }
 
-    public function importmap(?string $entryPoint = 'app', array $attributes = []): string
+    public function importmap(string|array $entryPoint, array $attributes = []): string
```

Comment:
this breaks existing apps that use [the default recipe](https://github.com/symfony/recipes/blob/d222415825b0e1605ec6b6271823a5894ed4cbc5/symfony/asset-mapper/6.3/manifest.json#L14)

is this desired?

Reply:
Removing support for the implement `app` entrypoint should go through a deprecation even if the component is experimental IMO due to this.

Reply:
and probably also the support of passing `null` then if we have a deprecation layer.

Reply:
good catch. Check the last commit. I've left `app` as the default (in part for BC). But in the recipe, I WOULD like to start people with `importmap('app')` because it's more explicit.